Readme.md

```diff
@@ -8,189 +8,137 @@ Docker images for computer vision and deep learning.
+  --build-arg with_pylon=true \
+  -f docker/moonbox/Dockerfile docker/moonbox
 ```
 
-| Build arg | Default | Description |
-|-|-|-|
-| `baseimage` | `moonvision/python-base` | Set the baseimage from which moonbox is build. |
-| `ffmpeg_from_docker` | `moonvision/custom-builds:ffmpeg-4.2.1` | The docker image from which FFmpeg will be installed. |
-| `pytorch_from_docker` | `moonvision/custom-builds:pytorch-1.11.0_torchvision-0.12.0` | The docker image from which PyTorch and Torchvision will be installed. |
-| `with_genicam` | `true` | Whether to install GeniCam harvesters. |
-| `with_pylon` | `true` | Whether to install Basler Pylon. |
-| `with_cuda` | `false` | Whether to install CUDA specific packages, (nothing at the moment). |
-
 #### Moonbox CUDA
 
 ```
-docker build -t moonvision/moonbox:cuda-latest -f docker/moonbox/Dockerfile --build-arg baseimage=moonvision/python-base:cuda-latest --build-arg with_cuda=true --build-arg ffmpeg_from_docker=moonvision/custom-builds:ffmpeg-cuda-4.2.1 --build-arg pytorch_from_docker=moonvision/custom-builds:pytorch-cuda-1.11.0_torchvision-0.12.0 docker/moonbox
+docker build \
+  -t moonvision/moonbox:cuda-latest \
+  --build-arg with_cuda=true \
+  --build-arg ffmpeg_from_docker=moonvision/custom-builds:ffmpeg-cuda-4.2.1 \
+  -f docker/moonbox/Dockerfile docker/moonbox
 ```
 
-| Build arg | Default | Description |
-|-|-|-|
-| `baseimage` | `moonvision/python-base:cuda-latest` | Set the baseimage from which moonbox is build. |
-| `ffmpeg_from_docker` | `moonvision/custom-builds:ffmpeg-cuda-4.2.1` | The docker image from which FFmpeg will be installed. |
-| `pytorch_from_docker` | `moonvision/custom-builds:pytorch-cuda-1.11.0_torchvision-0.12.0` | The docker image from which PyTorch and Torchvision will be installed. |
-| `with_genicam` | `false` | Whether to install GeniCam harvesters. |
-| `with_pylon` | `false` | Whether to install Basler Pylon. |
-| `with_cuda` | `true` | Whether to install CUDA specific packages, (nothing at the moment). |
-
-### builders
+## builders
 
 [Custom Builds on Docker Hub](https://hub.docker.com/r/moonvision/custom-builds)
 
-#### CMake
+### FFmpeg
 
-```
-docker build -t moonvision/custom-builds:cmake-latest -f docker/builders/cmake/Dockerfile docker/builders/cmake
-```
+A custom build of [FFmpeg](https://ffmpeg.org).
 
 | Build arg | Default | Description |
 |-|-|-|
-| `baseimage` | `ubuntu:20.04` | Set the image on which CMake will be built. |
-| `cmake_tag` | `v3.15.5` | The CMake version tag to build. |
+| `baseimage` | `mambaorg/micromamba:1` | Set the image in which FFmpeg will be built. |
+| `ffmpeg_version` | | The FFmpeg release version to build. |
+| `with_cuda` | `false` | Whether to build FFmpeg with Nvidia hardware acceleration.  |
 
-#### FFmpeg
+#### FFmpeg without CUDA
 
 ```
-docker build -t moonvision/custom-builds:ffmpeg-latest -f docker/builders/ffmpeg/Dockerfile --build-arg ffmpeg_version=4.2.1 docker/builders/ffmpeg
+docker build \
```

Comment:
There appears to be some weird special character or something in this command, it doesn't parse correctly.

<img width="601" alt="Screen Shot 2023-01-11 at 11 06 48" src="https://user-images.githubusercontent.com/1183303/211777624-b1417614-566a-4cbc-af17-a3d3c068090c.png">

Reply:
Or maybe a space after one of the backslaches

Reply:
Thanks, indeed a space invaded the commands. Should work now.